requirements.txt

```diff
@@ -42,7 +42,7 @@ citeproc-py==0.4.0
 boto3==1.4.7
 django-waffle==0.13.0
 pymongo==3.7.1
-progressbar==2.3
+git+https://github.com/coagulant/progressbar-python3@master
```

Comment:
It might be worth switching to https://github.com/tqdm/tqdm , which is Python 3 compatible and actively maintained.

Reply:
I agree, https://github.com/coagulant/progressbar-python3 is six years old and what about same import on `requirements/release.txt`?